helpers_test.go

```diff
@@ -82,9 +82,9 @@ func TestCmdWithFlags(t *testing.T) {
 				Usage: "List available needed patches for all CVE issues, or issues whose number matches the given string.",
 			},
 			cli.StringFlag{
-				Name:  "date",
+				Name:  "issues",
 				Value: "",
-				Usage: "Install patches issued up to, but not including, the specified date (YYYY-MM-DD).",
+				Usage: "doc",
```

Comment:
This is just used by the tests, we don't have to care. I didn't want to do a full copy&paste.